Euler1PS.js

```diff
@@ -0,0 +1,13 @@
+let sum = 0;
```

Comment:
This should be scoped to the function. What happens if I want to `findTheSum()` of two max numbers, say 500 and 800?

Reply:
Starting from above:

- Added a link to the problem in a comment
- By tests, I'm assuming you mean, picking criteria, and then running it through the program to make sure it produces the desired result?
- I added a prompt to allow user input
- I'm not sure how I would do a series of numbers. I image I would need to create an array that could hold the multiple inputs and then loop through the array.
   - I believe this would also go for finding two max numbers as well.